cmd/flags.go

```diff
@@ -70,6 +70,14 @@ func addOutputFormat(fs *pflag.FlagSet, outputFormat *util.EnumFlag) {
 	fs.VarP(outputFormat, "format", "f", "set output format")
 }
 
+func addListAnnotations(fs *pflag.FlagSet, value *bool) {
+	fs.BoolVarP(value, "annotations", "a", false, "list annotations")
+}
+
+func addAnnotationsFilter(fs *pflag.FlagSet, paths *[]string) {
```

Comment:
I just detected a bug with this flag: some invalid paths will cancel the filtering. Will fix.